tests/integration/test_docker.py

```diff
@@ -374,3 +374,12 @@ def test_list_images():
         assert len(image_list) > 0
         assert image_list[0].short_metadata
         assert image_list[0].short_metadata["Id"]
+
+
+def test_build_image():
+    with DockerBackend() as backend:
+        name = 'conu-test-image'
+        image = backend.ImageClass.build('/home/dhodovsk/repos/github.com/fedora-modularity/conu/tests/integration/data/greeter', tag=name)
```

Comment:
It would be nice if we can have some example dockerfile in the repo..;-)

Reply:
Heh, sorry this is just a typo, it should be of course `os.getcwd()` or similar, thanks for the catch :)